src/MorseL.Scaleout.Redis/RedisBackplane.cs

```diff
@@ -231,6 +231,7 @@ public RedisBackplane(IOptions<ConfigurationOptions> options)
             return RedisKeyGroupPrefix + group;
         }
 
+        internal IDictionary<string, Object> Connections => _connections;
         internal IDictionary<string, IDictionary<string, object>> Groups => _groups;

```

Comment:
Is it possible to type the objects as something more concrete? I haven't looked directly at usage yet but using object this pervasively seems less than ideal.

Reply:
This is me bastardizing ConcurrentDictionary for a concurrent hash. The values are always null.

Reply:
Ah 👍